openmrs/apps/registration/app.json

```diff
@@ -1,120 +1,136 @@
 {
-        "id": "bahmni.registration",
```

Comment:
@prashantsable1991 There's a lot of formatting changes here. Please revert and only commit the code changes you made.